pkg/utils/query/query.go

```diff
@@ -0,0 +1,338 @@
+/*
+The `query` utilities package contains helper functions for creating common KeyFilter values for filtering
+logs through `QueryKey`. A complete `ChainReader` implementation should include the following `ValueComparator`
+types.
+
+- AddressComparator
+- EventSigComparator
+- TopicComparator
+*/
+package query
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/types/query"
```

Comment:
Why do we need two separate `package query`s?

Reply:
Moved to `pkg/types/query`